cwms-data-api/src/main/java/cwms/cda/data/dao/BlobDao.java

```diff
@@ -1,14 +1,33 @@
 package cwms.cda.data.dao;
 
+import cwms.cda.api.errors.NotFoundException;
+import cwms.cda.data.dto.Blob;
+import java.io.InputStream;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
 import java.util.List;
 import java.util.Optional;
-
-import cwms.cda.data.dto.Blob;
+import java.util.function.Consumer;
+import kotlin.Triple;
+import org.jetbrains.annotations.NotNull;
 import org.jooq.DSLContext;
 import org.jooq.Record;
 import org.jooq.ResultQuery;
+import usace.cwms.db.jooq.codegen.packages.CWMS_TEXT_PACKAGE;
 
 public class BlobDao extends JooqDao<Blob> {
+
+    public static final String BLOB_WITH_OFFICE = "SELECT CWMS_MEDIA_TYPE.MEDIA_TYPE_ID, AT_BLOB.VALUE \n"
```

Comment:
Does the latest jooq codegen include the at_ tables?

If not, totally fine, if yes, we probably should them. 

Definitely okay with bypassing the view.

Reply:
Its a little unclear to me what your comment is asking for.   
Is there a view for Blobs?  Before these changes the BlobDao was already going to AT_BLOB and that table had all the pieces I needed so I didn't look into changing what was queried.
jOOQ doesn't handle Blob and Clob the way I want.  I don't want them mapped to String or byte[]. I want the actual Blob object so I can connect the stream directly into to javalin's result stream.

Reply:
I thought there was, but I might be wrong. but the JOOQ codegen *should* now also wrap the tables directly.

But if it doesn't behave correctly then don't worry about it. Though maybe a short comment explaining that for the next set of fools.